assets/workflowTemplate.yml

```diff
@@ -2,14 +2,21 @@ name: test
 
 on: workflow_dispatch
 
+# @NOTE If you have multiple CI Workflows you should
+# ensure that the CI processes are not running simultaneous analysis on contracts
+# Enable concurrency control below
+#concurrency:
+#  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request_target' && github.head_ref || github.ref }}
+#  cancel-in-progress: true
```

Comment:
This shouldn't matter - each workflow runs in isolation

Reply:
Workflows do not run in isolation with respect to the origin repository. Example: enabling `pull_request_target` affords PR's write permission access to the origin repo.

Reply:
What I mean is: Each workflow runs in an isolated job, with their own checked out version of the sources, so running different forge commands in different jobs should not have any negative effects, so there is no reason to add concurrency control, since none of the forge commands touch Github

Reply:
What I mean is: Each workflow runs in an isolated job, with their own checked out version of the sources, so running different forge commands in different jobs should not have any negative effects, so there is no reason to add concurrency control